purchases/src/test/java/com/revenuecat/purchases/SyncDispatcher.kt

```diff
@@ -0,0 +1,25 @@
+package com.revenuecat.purchases
+
+import com.revenuecat.purchases.common.Dispatcher
+import io.mockk.mockk
+import java.util.concurrent.RejectedExecutionException
+
+class SyncDispatcher : Dispatcher(mockk()) {
```

Comment:
Tests cannot import classes that live inside the tests folder of another module. That's why I made a copy to this module